README.md

```diff
@@ -232,6 +236,67 @@ To compile the same example (requires clang C compiler):
 Have fun!
 
 
+## Tests
+
+### Running all tests
+
+In the source folder, to run all tests across all backends use the following command:
+
+    make run_tests
+
+When testing is finished you will be presented a summary of succeeded, failed and skipped tests.
+
+To run the tests for one specific backend only use:
+
+    # for jvm backend
+    make run_tests_jvm
+
+    # for c backend
+    make run_tests_c
+
+    # for interpreter backend
+    make run_tests_int
+
```

Comment:
It should be documented that running all tests, or running all tests for a specific backend, will take quite a while nowadays and might be not what is desired (in many cases running specific tests is sufficient).